invenio_rdm_records/resources/config.py

```diff
@@ -371,6 +371,13 @@ class RDMDraftMediaFilesResourceConfig(FileResourceConfig, ConfiguratorMixin):
     ),
 }
 
+group_access_error_handlers = {
+    **deepcopy(RecordResourceConfig.error_handlers),
+    LookupError: create_error_handler(
+        HTTPJSONException(code=404, description="No grant found by given group id.")
```

Comment:
In general, are we making the error msg translatable?

Reply:
Yes, we should. For example this one is not only for the API, it could also be displayed in the UI. Thanks for noticing!